### DISCUSSION TITLE
Transaction date vs posted date

### DISCUSSION THREAD

**Discussion by adamroyle** [actualbudget/releases#157] [Q&A]
How do you handle differences in the transaction date (when you paid for an item) to the posted date (when it has processed and shows in your bank)?

Ideally I want to use the transaction date as that is when I made the purchase, but this makes it very hard to find mistakes as it causes the running balance to be incorrect. 

I have written a custom import script from my bank feed so I can extract the transaction date and use that, but it would be handy to have another field I can use for the posted date, which could be shown instead of the transaction date if the running balance column is shown. 

I am curious how others handle this situation currently, as I might be missing a logical solution to the issue.

**Comment by Deguello** [comment]
I don't use the running balance to compare to the bank when clearing transactions. I use the cleared total. It works fine.

**Comment by alehel** [comment]
I've actually been wondering about this myself. Would be nice if I could have both purchased date and cleared date.

**Comment by mrruz** [comment]
For normal transactions, I don't care about a few days difference and only reconcile to the ending balance of the bank to the cleared balance of my checkbook. But I also look at uncleared transactions that are really old and troubleshoot them separately. 

For transactions that I know are coming (like an insurance check, for example) but the vendor takes forever to clear, I book the transaction as uncleared in Actual, then when it clears weeks or months later, I just update the transaction date to the posting date so that it ties, but my budget still clears out the transactions properly.

**Comment by smathson** [comment]
I've only ever used the posted date for transactions and completely ignore any pending transactions until they post. I find it's easier to reconcile and track down any discrepancies if something goes wrong that way.
